tests/functional/db_cleanup/test_db_cleanup.py

```diff
@@ -0,0 +1,159 @@
+#
+# -----------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -----------------------------------------------------------------------------
+
+import json
+import multiprocessing
+import os
+import unittest
+import time
+
+from codeCheckerDBAccess_v6.ttypes import *
+
+from libtest import codechecker
+from libtest import env
+
+
+class TestDbCleanup(unittest.TestCase):
+
+    def setUp(self):
+        self.test_workspace = os.environ['TEST_WORKSPACE']
+
+        test_class = self.__class__.__name__
+        print('Running ' + test_class + ' tests in ' + self.test_workspace)
+
+        self.codechecker_cfg = env.import_codechecker_cfg(self.test_workspace)
+        self.test_dir = os.path.join(self.test_workspace, 'test_files')
+
+        try:
+            os.makedirs(self.test_dir)
+        except os.error:
+            # Directory already exists.
+            pass
+
+    def _create_test_dir(self):
```

Comment:
Private methods in python are start with two underscores.

Reply:
Single underscore also expresses the intention that the given entity is private like in JavaScript. The difference is that double underscore generates a mangled name for the function in the background which contains the class name as well. However in both cases the methods are available from outside, it's just more complicated in case of double underscores. I think it's up to the project convention which to use, I don't know if we have any. However I can rewrite it.

Reply:
I use double underscore. There is a [pep8 section](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables) about it but it does not mention the case if only one underscore is used.